config/initializers/session_store.rb

```diff
@@ -0,0 +1 @@
+Rails.application.config.session_store :active_record_store, :key => '_my_app_session'
```

Comment:
```suggestion
Rails.application.config.session_store :active_record_store, :key => 'civis_api_session'
```

Reply:
@manikandan0603 @sedhumadhavan19 - it should also have the environment mentioned, else it'll get confused when you login to both staging and production.